episode-11/main.py

```diff
@@ -36,16 +32,38 @@ def __init__(self, **args):
 
 		# pyglet stuff
 
-		pyglet.clock.schedule_interval(self.update, 1.0 / 10000)
+		pyglet.clock.schedule(self.update)
```

Comment:
Presumably because of vsync?

Reply:
Not really, its someone who suggested this as a fix for the flickering, so I added this

Reply:
Okay, I'll check it has the desired effect.